app/app.py

```diff
@@ -84,6 +84,16 @@ def create_json_response(ingred_paragraph, instr_paragraph, title, url):
     return json_response
 
 
+@app.route('/populate_training_example_from_url/', methods=['POST'])
+def populate_training_example_from_url():
+    password = request.form['password']
+
+    if password != "toy_password":
+        return {'error': 'Invalid password'}, 401
+
+    return {'message': 'Rows added to spreadsheet'}, 200
+
+
```

Comment:
This is the only change I should see in this PR.
The rest of the files/changes are relevant ONLY for your env.
you should stash them BEFORE committing.

why? 

think about the next step after I approve it.
1- We will merge it to the master branch
2- We will deploy to production, on a LINUX env that has a different configuration from yours - this will break the server.

So the lesson here is when you are doing a pull request commit only the minimum which is relevant for the ticket. nothing more.